common/reportMetrics.py

```diff
@@ -293,7 +293,7 @@ def send_data(metric_data_dict, file_path, chunk_serial_number):
         if response.status_code == 200:
             logger.info(str(len(bytearray(to_send_data_json))) + " bytes of data are reported.")
         else:
-            logger.info("Failed to send data.")
+            logger.info("Failed to send data. Response Code: " + str(response.status_code))
```

Comment:
Including the first few lines of the return message would be very helpful for debugging